src/librustc_middle/query/mod.rs

```diff
@@ -600,7 +648,9 @@ rustc_queries! {
 
         /// Per-body `region::ScopeTree`. The `DefId` should be the owner `DefId` for the body;
         /// in the case of closures, this will be redirected to the enclosing function.
-        query region_scope_tree(_: DefId) -> &'tcx region::ScopeTree {}
+        query region_scope_tree(def_id: DefId) -> &'tcx region::ScopeTree {
+            desc { |tcx| "getting scopes for `{}`", tcx.def_path_str(def_id) }
```

Comment:
`scopes` -> `drop scopes` (we should probably call them that consistently, too. unless I'm wrong about what they are)